src/sentry/static/sentry/app/views/projectPluginDetails.jsx

```diff
@@ -42,126 +54,107 @@ export default class ProjectPlugins extends AsyncView {
     });
   };
 
-  enable = () => {
-    this.toggleEnable(true);
-  };
-
-  disable = () => {
-    this.toggleEnable(false);
+  handleEnable = () => {
+    enablePlugin(this.props.params);
   };
 
   handleDisable = () => {
-    this.setState(prevState => ({
-      plugin: {
-        ...prevState.plugin,
-        enabled: false,
-      },
-    }));
+    disablePlugin(this.props.params);
   };
 
-  toggleEnable(shouldEnable) {
-    let method = shouldEnable ? 'POST' : 'DELETE';
+  // Enabled state is handled via PluginsStore and not via plugins detail
```

Comment:
this comment is slightly confusing, because the code seems like it sometimes defers to pluginDetails for enabled state

Reply:
It falls back to pluginDetails if for some reason it couldn't fetch the plugin from `PluginsStore`